llm/server.go

```diff
@@ -714,6 +699,39 @@ func (s *llmServer) Completion(ctx context.Context, req CompletionRequest, fn fu
 		"cache_prompt":      true,
 	}
 
+	if len(req.Format) > 0 {
```

Comment:
Moving this before the acquiring semaphore helps, but we should be checking these request fields before the model is submitted to the scheduler altogether (maybe in our http handler?). At this point in the code the model has already been reserved VRAM and scheduled to run.

cc @ParthSareen